common/src/main/resources/tachyon-default.properties

```diff
@@ -116,17 +114,17 @@ tachyon.user.heartbeat.interval.ms=1000
 tachyon.user.file.writetype.default=CACHE_THROUGH
 tachyon.user.file.tachyonstoragetype.default=STORE
 tachyon.user.file.understoragetype.default=SYNC_PERSIST
-tachyon.user.default.block.size.byte=512MB
+tachyon.user.block.size.bytes.default=512MB
 tachyon.user.quota.unit.bytes=8MB
 tachyon.user.file.buffer.bytes=1MB
-tachyon.user.remote.block.reader.class=tachyon.client.netty.NettyRemoteBlockReader
-tachyon.user.remote.block.writer.class=tachyon.client.netty.NettyRemoteBlockWriter
-tachyon.user.remote.read.buffer.size.byte=8MB
+tachyon.user.block.remote.reader.class=tachyon.client.netty.NettyRemoteBlockReader
+tachyon.user.block.remote.writer.class=tachyon.client.netty.NettyRemoteBlockWriter
+tachyon.user.block.remote.read.buffer.size.bytes=8MB
 tachyon.user.network.netty.worker.threads=0
 tachyon.user.network.netty.timeout.ms=3000
-tachyon.user.remote.block.worker.client.threads=10
-tachyon.user.local.block.worker.client.threads=10000
+tachyon.user.block.remote.worker.client.threads=10
```

Comment:
I slightly prefer `tachyon.remote.worker` and `tachyn.local.worker` as first, we have already been using prefix `tachyon.remote.worker` in properties like `tachyon.user.block.remote.reader.class` and `tachyon.user.block.remote.writer.class`, and they are related. Also `worker.client.threads` is really the identity of this property, I thought they make sense to be put together.